tests/testdata/vector_database_dataset.csv

```diff
@@ -0,0 +1,6 @@
+relevant
```

Comment:
nit, can probably trim some of the content in these guys so that it's not a large horizontal scrolling experience when viewing this CSV in github UI